### ISSUE TITLE
Normal videos

### ISSUE THREAD

**Issue by 89z** [jobber2955/GFBLD#4]
Do you know how to get normal videos, such as:

https://www.facebook.com/FromTheBasementPage/videos/309868367063220

**Comment by jobber2955** [comment]
Hi, According the URL you provide, I think It's the same way as ```GFBLD```. But I've only supported live video download.

You could try this manually:
1. Go to that video page, and select your wanted resolution, start playing(If the video auto-started during the loading stage, you should pause it, then jump to somewhere that haven't been buffered, then start playing)
2. After you eyes can tell you that this is truly the resolution you want, press ```F12```(on Chrome) to open ```DevTool```, then navigate to the ```Network``` tab, you should see lots of request beeing captured
3. After capturing for around 5~10s, pause the video, wait until there's no new request been captured. 
4. For easier to find, I suggest you click the ```Clear``` button just right above the search input. Then continue playing the video. At the very moment two new request were captured, pause the video.
5. You will find these two request are both ```.mp4```, click them, and copy the URL, remove the ```bytestart``` and ```byteend``` param, and enter it into your browser/downloader, you will get the video.

#### Remember, two request are both needed, they are audio and video. You should combined them yourself into a "real" video

**Comment by 89z** [comment]
Yes, I am aware of all that (in fact, MITM Proxy is a better option than the Network tab).

I am more referring to this:

https://github.com/jobber2955/GFBLD/blob/f05c9ba409933c7173da00bf691a31222ebbf113/gfbld/fetcher/fetcher.go#L103-L111

can it work with normal videos?

**Comment by jobber2955** [comment]
@89z sorry for late reply, busy on work.

MITM Proxy indeed is a better way for capturing, Chrome just the simpiest way to do this.

It should work since Facebook's live videos and normal video are actually the same and both are fetched from ```graphql```. Just change the two ```doc_id``` then you should be good to go
